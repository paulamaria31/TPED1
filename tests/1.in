7
OXVOXOXVO
OVVVXXVXO
VOXOOVOXX
OVVOOOOXO
OVXVVVXVO
XXOOOXXOO
XVVOXVOVX
