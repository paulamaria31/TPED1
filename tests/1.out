Tabuleiro 1 em andamento [X: (3,2)(1,3)]
Tabuleiro 2 em andamento [O: sem jogada mestre]
Tabuleiro 3 em andamento [X: (2,3)]
Tabuleiro 4 invalido
Tabuleiro 5 em andamento [proximo jogador indefinido]
Tabuleiro 6 deu velha
Tabuleiro 7 com vitoria [X]
